packages/rest/src/router/routing-table.ts

```diff
@@ -238,6 +240,31 @@ export interface ResolvedRoute extends RouteEntry {
   readonly schemas: SchemasObject;
 }
 
+export class StaticRoute implements RouteEntry {
+  public readonly verb: string = 'get';
+  public readonly path: string = '*';
+  public readonly spec: OperationObject = {responses: {}};
+
+  constructor(private _handler: Function) {}
+
+  updateBindings(requestContext: Context): void {
+    // no-op
+  }
+
+  async invokeHandler(
+    requestContext: Context,
+    args: OperationArgs,
+  ): Promise<OperationRetval> {
+    const req = await requestContext.get(RestBindings.Http.REQUEST);
+    const res = await requestContext.get(RestBindings.Http.RESPONSE);
```

Comment:
Let's use `context.request` and `context.response`, it should be faster than obtaining the values via Context bindings.